vpdq/cpp/hashing/bufferhasher.cpp

```diff
@@ -0,0 +1,60 @@
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include <pdq/cpp/common/pdqhashtypes.h>
+#include <pdq/cpp/downscaling/downscaling.h>
+#include <pdq/cpp/hashing/pdqhashing.h>
+#include <vpdq/cpp/hashing/bufferhasher.h>
+
+namespace facebook {
+namespace vpdq {
+namespace hashing {
+
+// ----------------------------------------------------------------
+bool PDQFrameBufferHasher::hashFrame(
```

Comment:
This function never returns false. Consider void instead?

Reply:
Update to return false if the frame height or width is smaller than MIN_HASHABLE_DIM